environments/staging/permissions.yml

```diff
@@ -168,15 +168,15 @@
       - 'CREATE_VERSION_ROLE'
   manager:
     - bare-template.aragonpm.eth:
```

Comment:
@facuspagnuolo For the sake of keeping this easy to maintain, it may be good idea to transfer the management of these roles to @izqui.